doc/requirements-docs.txt

```diff
@@ -1,40 +1,44 @@
 # these are most specified that usual, because Sphinx docs seem to be quite fragile
-Sphinx==4.4.0
+# limited <7 by sphinx-rtd-theme at the moment.
+# sphinx-rtd-theme requires docutils 0.18 but sphinx dropped that support in 6.0.0
+Sphinx==5.3.0
 
 # Parses Jupyter notebooks
-nbsphinx==0.8.8
+nbsphinx==0.9.2
 
 # Includes Jupyter notebooks in Sphinx source root
 nbsphinx-link==1.3.0
 
 # builds a HTML version of a Python script and puts it into a gallery
-sphinx-gallery==0.9.0
+sphinx-gallery==0.13.0
 
 # Read-The-Docs theme for Sphinx
-sphinx-rtd-theme==0.5.2
+sphinx-rtd-theme==1.2.2
 
 # allows us to more easily document our API
 sphinxcontrib-apidoc==0.3.0
 
-# generates OpenGraph metadata?
-sphinxext-opengraph==0.5.1
+# generates OpenGraph metadata to make good-looking cards on social media
+sphinxext-opengraph==0.8.2
 
-# supporting requirement documentation
-sphinxcontrib-needs==0.7.6
-sphinxcontrib-plantuml==0.23
+# supporting requirements traceability matrices for QA
+sphinx-needs==1.2.2
```

Comment:
this is where that `sphinx-data-viewer` dependency comes in, so this one fails too since that dep isn't 3.11 compatible

Reply:
ok this was a me-problem after all!